### PR TITLE
Added option to set resource type on service definition.

### PR DESCRIPTION
**Pull request by julianyus**
BreadcrumbTagHelper now tries to localize the titles.
Added sample of localization.

### CODE REVIEW

File: src/BreadcrumbOptions.cs [hunk 1]
```diff
@@ -113,6 +117,7 @@ public BreadcrumbOptions()
             FallbackTitleToMethodName = fallbackTitleToMethodName;
             RazorPagesRootDirectory = razorPagesRootDirectory;
             AriaLabel = ariaLabel;
+            ResourceType = null;
```
Comment (zHaytam) [inline src/BreadcrumbOptions.cs]: You are not using the argument `resourceType`

File: src/BreadcrumbOptions.cs [hunk 2]
```diff
@@ -81,6 +83,7 @@ public class BreadcrumbOptions
         /// A description that describes the navigation to screen reader users.
         /// </summary>
         public string AriaLabel { get; set; }
+        public Type ResourceType { get; set; }
```
Comment (zHaytam) [inline src/BreadcrumbOptions.cs:90]: Could you please add a little description about the option?

File: src/BreadcrumbOptions.cs (hunk 1 above)
Comment (julianyus) [reply src/BreadcrumbOptions.cs]: Fixed

File: src/BreadcrumbOptions.cs (hunk 2 above)
Comment (julianyus) [reply src/BreadcrumbOptions.cs:90]: Added :)

**Review by zHaytam — APPROVED** [review]

**Comment by zHaytam** [comment]
Thank you for the PR!

**Pull request merged**